### DISCUSSION TITLE
action.yml In root folder, but on submission it says no action.yml file

### DISCUSSION THREAD

**Discussion by nabeel-shakeel** [community/community#26167] [Actions]
<p>Hi,  I am running into problem, my action.yml file is in root folder of repository, but on submission to GitHub hackathon it still says <strong>no action.yml file</strong></p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/nabeel-shakeel">@nabeel-shakeel</a> ,</p>
<p>How did you use the action in your workflow file? Did you specify the location of the action.yaml? For example as below, use ‘./’ to specify the action directory, please make sure there is no typo here.</p>
<pre><code>- uses: ./ # Uses an action in the root directory
      id: hello
      with:
        who-to-greet: 'Mona the Octocat'
</code></pre>
<p>If you use ‘uses: {org}/{repo}<span class="mention">@tag</span>’ format in workflow, please check below:</p>
<p>Is this your repo: “<a href="https://github.com/nabeel-shakeel/graphite-docker-action" rel="nofollow noopener">nabeel-shakeel/graphite-docker-action</a>”? Please fix the action.yaml as below:</p>
<ol>
<li>
<p>Remove ‘-’ for ‘inputs’ parameters. The &lt;input_id&gt; must start with a letter or _ .  please refer to doc <a href="https://help.github.com/en/actions/building-actions/metadata-syntax-for-github-actions#inputsinput_id" rel="nofollow noopener">here</a>.</p>
</li>
<li>
<p>Use ‘image: “docker://graphiteapp/graphite-statsd:latest”’ or ‘Dockerfile’ for docker image, please refer to doc <a href="https://help.github.com/en/actions/building-actions/metadata-syntax-for-github-actions#examples-using-dockerfile-in-your-repository" rel="nofollow noopener">here</a>.</p>
</li>
</ol>
<p>Finally, the action.yaml file is like below:</p>
<pre><code>name: "Graphite"
description: "Run Docker Container of Graphite"
author: "Nabeel Shakeel(Nabeel_Shakeel@symantec.com)"

branding:
  icon: "shield"
  color: "gray-dark"

inputs:
  name:
    description: 'Container name'
    required: true
    default: graphite
  d:
    description: 'Run in backgroud'
    required: true
    default: ""

runs:
  using: "docker"
  image: "docker://graphiteapp/graphite-statsd:latest"
  args:
    - ${{ inputs.name }}
    - ${{ inputs.d }}
</code></pre>
<p>Hope it helps!</p>

**Comment by nabeel-shakeel** [comment]
<p>Hi <a class="mention" href="/weide-zhou">@weide-zhou</a> <br>
I am using {org}/{repo}<span class="mention">@tag</span> format to specify action.</p>
<p>Thanks for guiding on the parameters. </p>
